src/steamship/cli/cli.py

```diff
@@ -116,9 +124,87 @@ def deploy():
     # - Package content fails health check (ex. bad import) [Error caught while checking config object]
 
 
+@click.command()
+@click.option(
+    "--delete-data",
+    "-d",
+    is_flag=True,
+    default=False,
+    help="Delete all existing data by recreating the workspace.",
+)
+@click.option("--config", "-c", multiple=True)
+def test_instance(delete_data, config: List[str]):  # noqa: C901
```

Comment:
i think one thing that would be good is to somewhere provide the "why" of this command (both here and in docs). mainly i'd like to see something that captures how this differs from `temporary_workspace()` and why that's important.

it touches, iiuc, on the larger (potential) notion of cross-workspace artifact sharing, so capturing the intended use case clearly (near the code) would be good, imho.